### PR TITLE
Format code

### PR DESCRIPTION
**Pull request by devmotion**
I started going through the tutorials and updating them. But I thought it would be reasonable to apply a consistent code style first in a separate PR.

I think adopting a code style (I chose Blue style here since it is used in DynamicPPL, JuliaGPs, and some other Turing packages) will help to improve user experience and the quality of the tutorials. With https://github.com/domluna/JuliaFormatter.jl/pull/552 and https://github.com/domluna/JuliaFormatter.jl/pull/554, JuliaFormatter can also be used for jmd files which helps to enforce the code style.

The only issue was https://github.com/domluna/JuliaFormatter.jl/issues/555, caused by https://github.com/julia-vscode/CSTParser.jl/issues/328, which I worked around until a fix is available by disabling automatic formatting for the affected lines in the VI tutorial.

### CODE REVIEW

**Comment by devmotion** [comment]
Another worker misconfiguration, @staticfloat? Or is it an issue on our side?

File: tutorials/03-bayesian-neural-network/03_bayesian-neural-network.jmd [hunk 1]
```diff
@@ -244,42 +223,47 @@ function nn_forward(x, θ::AbstractVector, network_shape::AbstractVector)
...
-    preds = nn_forward(xs, θ, network_shape)
-    for i = 1:length(ts)
-        ts[i] ~ Bernoulli(preds[i])
+@model function bayes_nn_general(xs, ts, network_shape, num_params)
+    begin
+        θ ~ MvNormal(zeros(num_params), sig .* ones(num_params))
+        preds = nn_forward(xs, θ, network_shape)
+        for i in 1:length(ts)
+            ts[i] ~ Bernoulli(preds[i])
+        end
```
Comment (devmotion) [inline tutorials/03-bayesian-neural-network/03_bayesian-neural-network.jmd:239]:
```suggestion
    θ ~ MvNormal(zeros(num_params), sig .* ones(num_params))
    preds = nn_forward(xs, θ, network_shape)
    for i in 1:length(ts)
        ts[i] ~ Bernoulli(preds[i])
```
(Maybe good to wait with committing the fixes until the workers are fixed)

File: tutorials/07-poisson-regression/07_poisson-regression.jmd [hunk 2]
```diff
@@ -163,7 +183,20 @@ b2_exp = exp(mean(c1[:b2]))
...
+    "b1: ",
+    b1_exp,
+    " \n",
+    "b2: ",
+    b2_exp,
+    " \n",
+    "b3: ",
+    b3_exp,
+    " \n",
+)
```
Comment (devmotion) [inline tutorials/07-poisson-regression/07_poisson-regression.jmd:199]:
```suggestion
println("b0: ",  b0_exp)
println("b1: ",  b1_exp)
println("b2: ",  b2_exp)
println("b3: ",  b3_exp)
```

File: tutorials/07-poisson-regression/07_poisson-regression.jmd [hunk 3]
```diff
@@ -48,75 +48,97 @@ alcohol_meds = Poisson(theta_alcohol_meds)
...
+@model function poisson_regression(x, y, n, σ²)
+    begin
+        b0 ~ Normal(0, σ²)
+        b1 ~ Normal(0, σ²)
+        b2 ~ Normal(0, σ²)
+        b3 ~ Normal(0, σ²)
+        for i in 1:n
+            theta = b0 + b1 * x[i, 1] + b2 * x[i, 2] + b3 * x[i, 3]
+            y[i] ~ Poisson(exp(theta))
+        end
```
Comment (devmotion) [inline tutorials/07-poisson-regression/07_poisson-regression.jmd:141]:
```suggestion
    b0 ~ Normal(0, σ²)
    b1 ~ Normal(0, σ²)
    b2 ~ Normal(0, σ²)
    b3 ~ Normal(0, σ²)
    for i in 1:n
        theta = b0 + b1 * x[i, 1] + b2 * x[i, 2] + b3 * x[i, 3]
        y[i] ~ Poisson(exp(theta))
```

File: tutorials/04-hidden-markov-model/04_hidden-markov-model.jmd [hunk 4]
```diff
@@ -51,7 +82,6 @@ Simply put, this says that we expect state one to emit values in a Normally dist
 
 The priors on our transition matrix are noninformative, using `T[i] ~ Dirichlet(ones(K)/K)`. The Dirichlet prior used in this way assumes that the state is likely to change to any other state with equal probability. As we'll see, this transition matrix prior will be overwritten as we observe data.
 
-
 ```julia
 # Turing model definition.
 @model BayesHmm(y, K) = begin
```
Comment (devmotion) [inline tutorials/04-hidden-markov-model/04_hidden-markov-model.jmd:87]:
```suggestion
@model function BayesHmm(y, K)
```

**Comment by staticfloat** [comment]
> Another worker misconfiguration, @staticfloat? Or is it an issue on our side?

Yes, I forgot about those two hosts (The `exclusive` ones).  Fixed now.

**Review by rikhuijzer — APPROVED** [review]
I would also be okay with `margin=500` to avoid a lot of false-positive rewrites.

Anyway, this is great :+1:

File: tutorials/03-bayesian-neural-network/03_bayesian-neural-network.jmd [hunk 5]
```diff
@@ -107,13 +101,11 @@ sig = sqrt(1.0 / alpha)
 @model function bayes_nn(xs, ts)
     # Create the weight and bias vector.
     nn_params ~ MvNormal(zeros(20), sig .* ones(20))
```
Comment (devmotion) [inline tutorials/03-bayesian-neural-network/03_bayesian-neural-network.jmd:103]:
```suggestion
    nn_params ~ MvNormal(zeros(20), sig .* ones(20))

```

File: tutorials/03-bayesian-neural-network/03_bayesian-neural-network.jmd [hunk 6]
```diff
@@ -107,13 +101,11 @@ sig = sqrt(1.0 / alpha)
...
-    
     # Calculate predictions for the inputs given the weights
     # and biases in theta.
     preds = nn_forward(xs, nn_params)
```
Comment (devmotion) [inline tutorials/03-bayesian-neural-network/03_bayesian-neural-network.jmd:106]:
```suggestion
    preds = nn_forward(xs, nn_params)

```

**Comment by devmotion** [comment]
Yes. I think I just have to rename one of the tutorials that were pushed to the master branch. The filenames are too long.

**Pull request merged**